### PR TITLE
Change notice link colors to white

### PR DESCRIPTION
**Pull request by actionnick**
@exchgr Could you take a look at this for me?

### CODE REVIEW

File: app/assets/stylesheets/legacy_assets/modules/_notifications.css.scss [hunk 1]
```diff
@@ -32,10 +32,10 @@
 
   .notice {
     a {
-      color: $primary-color;
+      color: white;
```
Comment (exchgr) [inline app/assets/stylesheets/legacy_assets/modules/_notifications.css.scss:35]: Is there a sass variable for a white or whitish color in this gem? I think it's preferred that we use that if it exists. If not, no worries.

Comment (actionnick) [inline app/assets/stylesheets/legacy_assets/modules/_notifications.css.scss:35]: I checked [here](https://github.com/actionnick/legacy_assets/blob/change_notice_link_color/app/assets/stylesheets/legacy_assets/globals/_variables.css.scss), didn't see anything.

Comment (exchgr) [inline app/assets/stylesheets/legacy_assets/modules/_notifications.css.scss:35]: Seems fine to me. :shipit:

**Comment by rossta** [comment]
:+1:

**Pull request merged**